src/recovery.cpp

```diff
@@ -29,6 +29,15 @@ void recovery_setup() {
   });
   dnsServer.start(53, "*", WiFi.softAPIP());  // DNS spoofing.
   dnsServer.setErrorReplyCode(DNSReplyCode::NoError);
+  webServer.on("/reset", HTTP_ANY, [](AsyncWebServerRequest *request) {
```

Comment:
This is janky. The OTA library I'm using doesn't allow customization whatsoever so let's just nuke *all* settings whenever we log into recovery mode.

Reply:
Lmao, appreciate the compliment. Made a new function in recovery called `nukeSettings()`.